cirq/google/engine/engine_sampler.py

```diff
@@ -64,3 +64,26 @@ def run_sweep(
     @property
     def engine(self) -> 'cirq.google.Engine':
         return self._engine
+
+
+def get_engine_sampler(processor_id: str, gate_set_name: str) \
+        -> 'cirq.google.QuantumEngineSampler':
+    """Get an EngineSampler assuming some sensible defaults.
+
+    This requires an environment variable GOOGLE_GLOUD_PROJECT to be
+    set for the Engine project_id. We also assume you are using ProtoVersion.V2.
+
+    Args:
+        processor_id: Engine processor ID (from cloud console)
+        gate_set_name: One of ['sqrt-iswap', 'syc'].
+            See `cirq.google.NAMED_GATESETS`.
+    """
+    try:
+        gate_set = gate_sets.NAMED_GATESETS[gate_set_name]
+    except KeyError:
+        raise ValueError(f"Please use one of the following gateset names: "
+                         f"{sorted(gate_sets.NAMED_GATESETS.keys())}")
+
+    return engine.Engine(project_id=os.environ['GOOGLE_CLOUD_PROJECT'],
```

Comment:
FWIW, in situations I like to add an arg so the environment dict can be injected in tests, rather than monkey-patching:
```python
def get_engine_sampler(processor_id: str, gate_set_name: str, env: Dict[str, str] = os.environ):
    ...
    return engine.Engine(project_id=env['GOOGLE_CLOUD_PROJECT'], ...)

# in tests:
get_engine_sampler('foo', 'gates', {'GOOGLE_CLOUD_PROJECT': 'my-project'})
```
Some people are not too opposed to monkey-patching, but it always scares me :-)

Reply:
1. Actually yeah, I should add an argument that defaults to the environment variable
2. I'm going to leave the monkey patching test in as well. I think the primary utility of this method is dispatching to the environment variable for project id so people can share code that doesn't have their personal project id hardcoded

Reply:
added!